astropy/io/ascii/ui.py

```diff
@@ -1059,10 +1058,10 @@ def write(
         # behavior is for Python to translate \r\n (which we write because
         # of os.linesep) into \r\r\n. Specifying newline='' disables any
         # auto-translation.
-        output = open(output, "w", newline="")
-        output.write(outstr)
-        output.write(os.linesep)
-        output.close()
+        with open(output, "w", newline="") as output:
+            output.write(outstr)
+            output.write(os.linesep)
+            output.close()
```

Comment:
The `.close()` is not needed with the context manager.

Reply:
Thanks for catching this. Fixed.